src/vm/callingconvention.h

```diff
@@ -210,7 +210,11 @@ struct TransitionBlock
             return argLocDescForStructInRegs->m_cFloatReg > 0;
         }
     #endif        
+    #ifdef ARM_SOFTFP
+        return (FALSE);
```

Comment:
This should not be needed. The offset should never be < 0 on ARM_SOFTFP. If you can see the offset being negative with ARM_SOFTFP, then it needs to be fixed at the place where the offset is generated.